.gitignore

```diff
@@ -69,3 +69,4 @@ contracts-exposed
 .last_confs
 certora_*
 .zip-output-url.txt
+.worktree
```

Comment:
We need to add a comment for this, every other ignored file is in its own category block and this one doesn't belong to `# Certora`

Reply:
That was me, let me remove this.